shop_app.rb

```diff
@@ -0,0 +1,47 @@
+require "bundler/setup"
+require "sinatra/base"
+require "sinatra/reloader"
+
+Dir["./lib/**/*.rb"].each { |ruby_file| require ruby_file }
+require "./db/seeds.rb"
+
+module Shop
+  class App < Sinatra::Base
```

Comment:
Sorry, not sure if I understand correctly. I have a 'shop_app.rb' file in 'shop' directory. Do you mean renaming it? If so, shouldn't it be named 'shop.rb' after 'module Shop' instead?